torch/_inductor/codegen/wrapper.py

```diff
@@ -226,9 +226,16 @@ def writeline(line: str, example_grid: Optional[str] = None):
             assert len(grids) > 1
             assert len(grids) == len(configs)
             seen = set()
-            for grid, c in zip(grids, configs):
+            # sort the configs from the largest # of kwargs to the smallest to
+            # emit the grids in the order of (approximately) decreasing specificity
+            # TODO(aakhundov): the sorting below is generally not sufficient, so
+            # maybe we'll need to restrict the supported cases to identical kwarg
+            # names in all autotuning configs.
```

Comment:
@oulgen re. this, I think, we can start with this version (which is better than what we had before) and see if we bump into issues. My understanding is that using non-uniform configs (by kwargs) is pretty non-standard and I don't want to break existing use cases by enforcing uniformity. wdyt?